gdcdictionary/schemas/follow_up.yaml

```diff
@@ -1185,6 +1377,9 @@ properties:
     maximum: 32872
     minimum: -32872
 
+  days_to_first_event:
```

Comment:
Do this and the other new properties need a `$ref`?

Reply:
Yes, this should be resolved by now.